### ISSUE TITLE
Test Page doesn't work on IE

### ISSUE THREAD

**Issue by patrickzurek** [eXtensibleCatalog/xcncip2toolkit#112]
JIRA issue created by: rcook
Originally opened: 2012-06-27 12:55 PM

Issue body:

http://code.google.com/p/xcncip2toolkit/issues/detail?id=32   See this link for image attachment.

Reported by project member rcook@library.rochester.edu, Jul 27, 2011

It seems the NCIP test page works in Firefox but not IE.

http://128.151.244.132:8080/ncipv2_v_0_1_1/ - one of Rochester test pages (currently up).

When I try the above url in IE, the response box hangs with "Got something" but no content and the page gets an "Error on page".

Delete comment
Comment 1 by project member pzurek@uillinois.edu, Jul 27, 2011

You can use transformNode instead of XSLTProcessor like so:

var myXMLHTTPRequest = new XMLHttpRequest();
myXMLHTTPRequest.open("GET", "./stylesheet.xsl", false);
myXMLHTTPRequest.send(null);
xslStylesheet = myXMLHTTPRequest.responseXML;

var output = req.responseXML.transformNode(xslStylesheet);
document.ajax.msgLine.value=output;

However, the output looks terrible and it isn't formatted in the hierarchical way it's presented in Firefox so I don't even know if it's worth making the change in the core.  People might use IE and think that's the way it should look rather than have it fail and switch to Firefox.  I've attached a screenshot.

Cc: rcook@library.rochester.edu
Delete comment
Comment 2 by project member pzurek@uillinois.edu, Jul 27, 2011

(No comment was entered for this change.)

	IEoutput.JPG
146 KB   View   Download 	
Delete comment
Comment 3 by project member bodfishj@oclc.org, Mar 16, 2012

I propose fixing this early in the 1.2 release cycle, after we correct the maven version # ( issue #40 ) and move to Java 1.7 ( issue #43 ).

Labels: Release-1.2

**Comment by patrickzurek** [comment]
JIRA Coment by user: bodfishj
JIRA Timestamp: 2012-11-29 04:03 PM

Comment body:

This was fixed by Adam Constabaris, when he converted the test page to use jquery.

**Comment by patrickzurek** [comment]
Issue resolved: 2012-11-29 04:03 PM
